.circleci/config.yml

```diff
@@ -34,6 +34,13 @@ jobs:
                     - create_zip_upload_to_s3:
                         lambdaFolder: "webhook-testing-bdf90c40-b8ea-409b-951a-3ee800a00dda/deployment"
                         s3BucketFolder: "webhookTesting"
+    tests:
+        docker:
+            - image: circleci/buildpack-deps:18.04-browsers
```

Comment:
This seems an overkill image, based on the name, but on the other hand, the builds are under a minute, so it's fine.

Reply:
We use -browsers in ui2 because of karma